sdk/src/signer/keypair.rs

```diff
@@ -25,6 +25,9 @@ use {
 pub struct Keypair(ed25519_dalek::Keypair);
 
 impl Keypair {
+    /// Can be used for generating a Keypair without a dependency on `rand` types
```

Comment:
This comment might be too specific, but it shows why I needed it

Reply:
Even better, will do!

Reply:
Done in [db48acc](https://github.com/solana-labs/solana/pull/32871/commits/db48acc9924e47d9495d4b285f998188f673ef8c)